sunpy/map/tests/test_plotting.py

```diff
@@ -92,10 +93,26 @@ def test_rectangle_aia171(aia171_test_map):
     aia171_test_map.plot()
     bottom_left = SkyCoord(
         0 * u.arcsec, 0 * u.arcsec, frame=aia171_test_map.coordinate_frame)
+    top_right = SkyCoord(
+        100 * u.arcsec, 100 * u.arcsec, frame=aia171_test_map.coordinate_frame)
     w = 100 * u.arcsec
     h = 100 * u.arcsec
     aia171_test_map.draw_rectangle(bottom_left, w, h)
```

Comment:
Will this line not trigger a deprecation warning now? Should this be updated to use `top_right`?

Reply:
This was for a figure test. I'll change it.